### ISSUE TITLE
RPM dependency issue

### ISSUE THREAD

**Issue by mamoep** [go-graphite/go-carbon#144]
Installing updates on RHEL fails when the go-carbon RPM package is installed.

```
yum update
...
Error: Package: go-carbon-0.9.1-1.el6.x86_64 (@/go-carbon-0.9.1-1.el6.x86_64)
           Requires: /etc/init.d/functions
           Removing: initscripts-9.03.53-1.el6_8.1.x86_64 (@repo)
               Not found
           Updated By: initscripts-9.03.53-1.el6_8.2.x86_64 (repo)
               Not found
...
```

**Comment by halcyonardency** [comment]
Also broken with 0.8.1:
```
--> Processing Dependency: /etc/init.d/functions for package: go-carbon-0.8.1-1.el6.x86_64
--> Finished Dependency Resolution
Error: Package: go-carbon-0.8.1-1.el6.x86_64 (@/go-carbon-0.8.1-1.el6.x86_64)
           Requires: /etc/init.d/functions
           Removing: initscripts-9.49.30-1.el7_2.3.x86_64 (@xxx-centos-production-updates-v7-64bit)
               Not found
           Updated By: initscripts-9.49.37-1.el7.x86_64 (centos-base-prod)
               Not found
```

**Comment by lomik** [comment]
related commit https://github.com/lomik/go-carbon/commit/c4d3c6fd2f1fb49777c37faaac4709b89297cb7b

Will be deleted in the next release

**Comment by mamoep** [comment]
Instead of just deleting the line, the dependency should be changed to "initscripts", which provides /etc/init.d/functions

**Comment by lomik** [comment]
New init script has been released in 0.10. It not depends on /etc/init.d/functions
